arangod/VocBase/Methods/Databases.cpp

```diff
@@ -186,11 +186,15 @@ Result Databases::createCoordinator(CreateDatabaseInfo const& info) {
     return Result(TRI_ERROR_ARANGO_DATABASE_NAME_INVALID);
   }
 
+  LOG_TOPIC("56372", DEBUG, Logger::CLUSTER) << "createDatabase on coordinator: Starting, name: " << info.getName();
```

Comment:
I suggest that this log entry (and the following ones that are added in this PR) are downgraded to log level TRACE.

Reply:
Again, if you don't mind please collect theses as suggestions for a PR, that could be changed everywhere. We use the DEBUG on client support cases and TRACE holds way too much. This is good information in DEBUG level